src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/Predicates/PredicateQuery.cs

```diff
@@ -121,15 +127,15 @@ public string GetColumnName(string propertyPath)
                     {
                         // Switch the given alias in the path with the mapped alias.
                         // aliasPart.alias -> AliasPartIndex.Alias
-                        return Dialect.QuoteForTableName($"{tableAlias}") + "." + Dialect.QuoteForColumnName(columnName);
+                        return Dialect.QuoteForTableName($"{tableAlias}", _yesSqlConfiguration.Schema) + "." + Dialect.QuoteForColumnName(columnName);
```

Comment:
If `tablePrefix` is not used here, it's probably because this is just an alias, and then schema can/should be null.